db/migrate/20220507150428_add_gym_leader.rb

```diff
@@ -0,0 +1,5 @@
+class AddGymLeader < ActiveRecord::Migration[6.1]
+  def change
+    add_column :users, :is_gym_leader, :boolean
```

Comment:
Why do we need an additional status on users?

A gym leader is 'false' when: **Pokemon.all.where(user: @user) == 0**
It is true otherwise. I think that it creates a door for bugs later on. if we add a field that doesn't contain 'input-data'. 

What do you think?